packages/ui/v2/core/SettingsToggle.tsx

```diff
@@ -0,0 +1,57 @@
+import { useAutoAnimate } from "@formkit/auto-animate/react";
```

Comment:
Does this have to be in v2?

Reply:
No it doesn't - but I believe it should live here until we fully migrate everything from V2 -> v1 files so we don't accidentally remove anything we are meant to in the clean up